src/framework/MOM_io.F90

```diff
@@ -5,6 +5,7 @@ module MOM_io
 
 use MOM_array_transform,  only : allocate_rotated_array, rotate_array
 use MOM_array_transform,  only : rotate_array_pair, rotate_vector
+use MOM_coms,             only : sync_PEs
```

Comment:
I believe that this module use statement for sync_PEs is unnecessary.

Reply:
Thanks, this has been removed